Rename ordata to assets (#40)
personalrobotics/pr-ordata: README.md

# pr_assets
pr_assets is a repository of [OpenRAVE](http://openrave.org/) resources used by

> the [Personal Robotics Lab](https://personalrobotics.ri.cmu.edu/) in the

Reviewer: ```suggestion
the [Personal Robotics Lab](https://personalrobotics.cs.washington.edu/).
```

> [Robotics Institute](https://www.ri.cmu.edu/) at [Carnegie Mellon

Reviewer: ```suggestion
```

> University](http://www.cmu.edu/). Primarily, this includes a repository of

Reviewer: ```suggestion
Primarily, this includes a repository of
```

## Contributors

> pr_assets was developed by the [Personal Robotics Lab](https://personalrobotics.ri.cmu.edu)

Reviewer: ```suggestion
pr_assets was developed by the [Personal Robotics Lab](https://personalrobotics.cs.washington.edu).
```

> in the [Robotics Institute](http://ri.cmu.edu) at [Carnegie Mellon University](http://www.cmu.edu).

Reviewer: ```suggestion
```
